vividus-engine/src/main/java/org/vividus/batch/BatchConfiguration.java

```diff
@@ -130,4 +152,24 @@ private List<String> convertToList(String list)
                      .map(String::strip)
                      .collect(Collectors.toList());
     }
+
+    private static final class StoryExecutionConfiguration
+    {
+        private boolean failFast;
+
+        private void setFailFast(boolean failFast)
+        {
+            this.failFast = failFast;
+        }
+    }
+
+    private static final class ScenarioExecutionConfiguration
+    {
+        private boolean failFast;
+
+        private void setFailFast(boolean failFast)
+        {
+            this.failFast = failFast;
+        }
+    }
```

Comment:
minor question, do we need both StoryExecutionConfiguration and ScenarioExecutionConfiguration? it look like we could use one class with generic name

Reply:
Based on the issue https://github.com/vividus-framework/vividus/issues/3416 this value will be inserted into the story config